docs/package/gluon-web-network.rst

```diff
@@ -0,0 +1,19 @@
+gluon-web-network
+=================
+
+The package *gluon-web-network* is part of the :ref:`Feature Flag <user-site-feature-flags>` web-advanced.
```

Comment:
```suggestion
The package *gluon-web-network* is part of :ref:`Feature Flag <user-site-feature-flags>` web-advanced.
```

Reply:
I think this is correct before in the way that you say:
The scene is part of the Movie XYZ
and not
The scene is part of Movie XYZ?

Though one probably can say both..?

Reply:
@maurerle 
![image](https://github.com/freifunk-gluon/gluon/assets/2787581/a534db16-8499-4cad-a4f6-03f29b783377)